src/main/webapp/dashboard-manager.js

```diff
@@ -102,7 +125,8 @@ class DashboardManager {
     let result = google.visualization.data.group(
         filtered,
         [depth - 1],
-        [{'column': filtered.getNumberOfColumns() - 1, 'aggregation': google.visualization.data.sum, 'type': 'number'}]);
+                    // Devices count is second to last column
```

Comment:
The indent of the comment looks weird. If you decide to make `getNumberOfColumns() - 2` a constant as mentioned in a previous comment, then this comment can be removed.